opennms-dao/src/main/java/org/opennms/netmgt/dao/hibernate/DistPollerDaoHibernate.java

```diff
@@ -42,7 +42,8 @@ public DistPollerDaoHibernate() {
 
     @Override
     public OnmsDistPoller whoami() {
-        // Return the OnmsDistPoller with the default UUID
-        return get(DEFAULT_DIST_POLLER_ID);
```

Comment:
IMO, the whole `DEFAULT_DIST_POLLER_ID` should be removed (or at least moved to the test scope).

Reply:
That's a good point - would make sure there are no longer any references to it.